pkg/manager/getconfig.go

```diff
@@ -47,6 +50,33 @@ func (m *Manager) GetTargetConfig(target string, configname string, path string,
 				fmt.Errorf("No Configuration found for %s", configname)
 		}
 	}
+
+	readOnlyPaths, ok := m.ModelRegistry.ModelReadOnlyPaths[utils.ToModelName(config.Type, config.Version)]
+
+	if !ok {
+		log.Warningf("Cannot check for Read Only paths for %s %s because "+
+			"Model Plugin not available - continuing", config.Type, config.Version)
+	} else {
+		//TODO does this account for list values?
+		log.Info("path ", path)
+		log.Info("readonlypaths ", readOnlyPaths)
+		log.Info("Op state cache", m.OperationalStateCache[topocache.ID(target)])
+		_, ok := readOnlyPaths[path]
+		if ok {
+			value, ok := m.OperationalStateCache[topocache.ID(target)][path]
+			if ok {
+				cachedValues := make([]*change.ConfigValue, 0)
+				cachedValues = append(cachedValues, &change.ConfigValue{
+					Path:       path,
+					TypedValue: *value,
+				})
+				return cachedValues, nil
```

Comment:
return here? what about the config stuff below?

Reply:
good catch, thanks.